ihmc-high-level-behaviors/src/main/java/us/ihmc/PerceptionAndAutonomyProcess.java

```diff
@@ -614,6 +687,7 @@ private void initializeDependencyGraph(ROS2PublishSubscribeAPI ros2)
       zedColorDemandNode.addDependents(zedPointCloudDemandNode, centerposeDemandNode, ballDetectionDemandNode);
       planarRegionsDemandNode.addDependents(yoloZEDDemandNode); // Planar region used for door detection
       realsenseDemandNode.addDependents(yoloRealsenseDemandNode);
+      realsenseDemandNode.addDependents(heightMapDemandNode);
```

Comment:
@TomaszTB Double check me here

Reply:
Looks correct to me! When the `heightMapDemandNode` is demanded it will also demand the realsense node.